specification.md

```diff
@@ -560,6 +560,15 @@ in the future such that old Agents automatically report that they don't
 support the new capability.
 This field MUST be always set.
 
+This field MUST be set in the first AgentToServer sent by the Agent and MAY be
```

Comment:
Should we merge these sections somehow? I feel like the second paragraph overrides the first a bit.